File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/CodiciAnagrafeValori.ts
import { Optional } from '@angular/core';
import { CodiceAnagrafe } from './CodiceAnagrafe';
import { IntervalloTemporale } from './IntervalloTemporale';

export class CodiciAnagrafeValori {
    codiceAnagrafe: CodiceAnagrafe;
    valore: string;
    validita: IntervalloTemporale;
}

export class CodiciAnagrafeValoriChiave extends CodiciAnagrafeValori {
    chiave: number;
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/CodiciAnagrafeValori.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class CodiciAnagrafeValori
    {
        public CodiceAnagrafe codiceAnagrafe { get; set; }

        public string valore { get; set; }

        public IntervalloTemporale validita { get; set; }

        public CodiciAnagrafeValori()
        {
        }
    }
}
